upgrade_tests/test_scenarios/test_user.py

```diff
@@ -0,0 +1,104 @@
+"""Test for User related Upgrade Scenario's
+
+:Requirement: Upgraded Satellite
+
+:CaseAutomation: Automated
+
+:CaseLevel: Acceptance
+
+:CaseComponent: API
+
+:TestType: Functional
+
+:CaseImportance: High
+
+:Upstream: No
+"""
+
+from upgrade_tests import pre_upgrade, post_upgrade
+
+
+class scenario_positive_create_sshkey_in_existing_users:
+    """SSH Key can be created in existing user post upgrade
+
+    :id: e4338daa-272a-42e3-be45-77e1caea607f
```

Comment:
We need to make sure this approach will work with polarion

Reply:
Yeah, We will propose this to @elyezer, if that doesn't work then we can update accordingly. But as of now, we can get this in. So that folks can add similar tests of their features

Reply:
This won't work. I talked with @jyejare and recommended creating a custom script to handle the way you decided to design the tests.

This is also using the test libraries in a way they were not designed. As betelgeuse each test function/method is mapped to one test case in polarion because each test function/method generates one entry when generating a junit file. So each entry provides a result for a single test case on Polarion.

The way these tests are being designed, it could have multiple results on the junit mapping to a single test case on Polarion. Since this is a very specific use case a custom tool needs to be created to both generate the XML for the test case importer considering this design and also to proper interpret the jUnit result generated by the Python tools to update a test result on a test run.

Not related nitpick: http://pep8.org/#class-names